### DISCUSSION TITLE
Zerotier

### DISCUSSION THREAD

**Discussion by indielab** [WLAN-Pi/feedback#1] [General Inquiries] [answered]
Has anyone tried connecting to Zerotier? It's installed on my WlanPi and connected to my Zerotier network. I opened port 9993 and verified that Zerotier-one is listening but I cannot connect to the WlanPi using it's Zerotier network address. I haven't had this problem with other devices and have exhausted all the troubleshooting avenues I can think of. Any suggestions?

**Comment by joshschmelzle** [comment]
Yes, several of us on the development team use ZT.

Are you sure both the WLAN Pi and the device you are using to reach the WLAN Pi are connected to the same ZT network?

I would start by verifying directly on WLAN Pi that it is ONLINE and connected to your desired ZT network.

* `sudo zerotier-cli info` - is ZT ONLINE?
* `sudo zerotier-cli listnetworks` - is ZT connected to a network? Is the status OK?

If both of those are good, can you ping the ZT address on the WLAN Pi from another device connected to your ZT network?

If both the WLAN Pi and your device is connected to the same ZT network, perhaps there is some MTU issue going on.

**Comment by indielab** [comment, marked as answer]
Thanks, Josh. I discovered that I could ping ZT devices outside my LAN but not inside (was not like this before). After some digging I set "Allow assignment of global IPs" which made everything work again. Still learning the quirks of ZT I guess but great to be able to use it on the WLAN Pi!
